xformers/__init__.py

```diff
@@ -10,8 +10,11 @@
 # Please update the doc version in docs/source/conf.py as well.
 __version__ = "0.0.12.dev"
 
-_is_sparse_available = True
-_is_triton_available = torch.cuda.is_available()
+_is_sparse_available: bool = True
+_is_triton_available: bool = torch.cuda.is_available()
+
+# Set to true to utilize functorch
+_is_functorch_available: bool = False
```

Comment:
What is the desired way to let users have the nvfuser path enabled? Will they need to clone and modify xformers for that?

Reply:
I think currently we have it as something that users will need to modify. I haven't looked into it a great deal, perhaps users who haven't cloned xformers can do something like `python -c "import xformers; xformers._is_functorch_available=True;`